sklearn/ensemble/_weight_boosting.py

```diff
@@ -328,6 +328,13 @@ class AdaBoostClassifier(ClassifierMixin, BaseWeightBoosting):
         Thus, it is only used when `base_estimator` exposes a `random_state`.
         Pass an int for reproducible output across multiple function calls.
         See :term:`Glossary <random_state>`.
+    n_samples : int, default = None
+        If not set to None, the Classifier will pick n random samples at each
+        boosting iteration using the resample function. 
+    replace : boolean, default = True
```

Comment:
I don't know if it is best to call this `replace`, `with_replacement` or just `bootstrap`.